website/client/src/components/Routes/PrivateRouteAdmin.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { IS_ADMIN } from '../../constant/constant';
+import { Redirect, Route } from 'react-router-dom';
+
+const PrivateRouteAdmin = ({ component: Component, ...rest }) =>
+    (
+        <Route {...rest} render={
+            (props) =>
+                (
+                    rest.user ? (
+                        rest.user.role === IS_ADMIN ? <Component /> : <Redirect to='/admin/login' />
```

Comment:
Where do you get "user" prop from?
A little problem with naming a variable  "IS_ADMIN", is this value is a boolean type or a string.

Reply:
I get in Routes component and send to PrivateRoute as props. IS_ADMIN is a constant integer number

Reply:
Ok I got it, why not connect redux in the PrivateRoute so you don't have to pass "user" every time you have new private routes.
Just name it "ADMIN", "is" or "should" is the best-practice for prefix name with a boolean variable. such as "isOpen" for the modal.